### PR TITLE
Audit documentation now calls out that pc and pr are both supported.

### PR DESCRIPTION
**Pull request by nkolev92**
Some of the documentation was altered to match the property style from restore/pack and to remove redundant/repetitive wording.

Fixes https://github.com/NuGet/docs.microsoft.com-nuget/issues/3247

### CODE REVIEW

**Comment by nkolev92** [comment]
@NuGet/nuget-client - Can I please get a review.

File: docs/concepts/Auditing-Packages.md [hunk 1]
```diff
@@ -79,30 +74,16 @@ On NuGet.org, you can navigate to the package details page and click `Report pac
...
-Possible values are `low`, `moderate`, `high`, and `critical`.
-For example if you only want to see `moderate`, `high`, and `critical` advisories, you can set the following:
+Audit can be configured via MSBuild properties in a `.csproj` or MSBuild file being evaluated as part of your project.
+We recommend that audit is configured at a repository level.
 
-```xml
-<NuGetAuditLevel>moderate</NuGetAuditLevel>
-```
+| MSBuild Property | Default | Possible values | Notes |
+|------------------|------_--|-----------------|-------|
```
Comment (Nigusu-Allehu) [inline docs/concepts/Auditing-Packages.md:83]: - instead of _ ?

**Review by Nigusu-Allehu — APPROVED** [review]
LGTM, just a minor comment

File: docs/concepts/Auditing-Packages.md (hunk 1 above)
Comment (nkolev92) [reply docs/concepts/Auditing-Packages.md:83]:
```suggestion
|------------------|---------|-----------------|-------|
```

**Pull request merged**